handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+# MongoDB
+
+MongoDB is used by the majority of SystemLink services as the primary datastore. MongoDB Community Edition is included in the default installation of SystemLink will launch when the SystemLink server boots.
+
+!!!note
+    By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server. This is referred to as a *single node* deployment. To improve the reliability of data storage and scale to larger volumes of data, remote data stores are recommended for all production deployments. This is referred to as a *multi node* deployment.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2021R1 or greater. Refer to Installing and Configuring SystemLink Server and Clients for the basics of setting up a SystemLink server.
+- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account with a user with the `dbAdmin` or similar role. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details.
+
+### SystemLink with Single Node vs Multi Node MongoDB Deployments
```

Comment:
I love this table! Anything we can do to ease the constant decisions users have to make will always be helpful.

I can see this part living in SPD in the future, since it's ostensibly helping people choose whether they even need to refer to the rest of this documentation (which covers the multi-node use cases only). We could point people here for the multi-node use cases if we still consider them advanced/niche.

Reply:
I totally agree. I'm thinking about having versions of this table in the other datastore documents to show the relationship. In an omni-bus SPD version I would encourage showing the single node version and have the multi node versions show all the different data stores on separate servers all at once rather than just showing one data store at a time on a separate server (this would be reflective of our recommended approach). Said another way we probably don't need a table with 15+ rows. I bet we could get the point across in 5 or 6.